.appveyor.yml

```diff
@@ -0,0 +1,51 @@
+os:
+  - Visual Studio 2017
+
+# do not build on tags
+#skip_tags: true
+
+# clone only the top level commit
+shallow_clone: true
+
+# branches to build: all but pages
+branches:
+  except:
+    - gh-pages
```

Comment:
copy/paste?

Reply:
I don't see a copy-paste error here, but I did remove the comment since it wasn't adding much.  Let me know if there's still a bug somewhere I'm just not seeing.

Reply:
We don't have a `gh-pages` branch and I don't think we want any branch exclusions at the moment. I would remove the whole `branches` tree.

Reply:
Sure, no problem, I've removed this whole section.  If we ever do add github pages for the CTS we should add it as a branch exclusion, though.